packages/gestalt-charts/src/ChartGraph.js

```diff
@@ -0,0 +1,586 @@
+                    orientation="bottom"
+                    {...(isTimeSeries ? { scale: 'time' } : {})}
+                    style={FONT_STYLE_CATEGORIES}
+                    tickFormatter={
+                      isTimeSeries
+                        ? tickFormatter?.xAxisBottom || tickFormatter?.timeseries
+                        : (value: string) => labelMap?.[value] || value
+                    }
+                    tickLine={false}
+                    type={isTimeSeries ? 'number' : 'category'}
+                    // DO NOT SET xAxisId here
+                  />
+                  <YAxis
+                    axisLine={false}
+                    domain={Array.isArray(range) ? range : range.yAxisLeft}
+                    orientation="left"
+                    style={FONT_STYLE_VALUES}
+                    tickLine={false}
+                    tickCount={tickCount}
+                    yAxisId="left"
+                    tickFormatter={tickFormatter?.yAxisLeft}
+                  />
+                </Fragment>
+              ) : null}
+              {isHorizontalBiaxial ? (
+                <YAxis
+                  axisLine={false}
+                  domain={Array.isArray(range) ? range : range.yAxisLeft}
+                  orientation="right"
+                  style={FONT_STYLE_VALUES}
+                  tickLine={false}
+                  tickCount={tickCount}
+                  yAxisId="right"
+                  tickFormatter={tickFormatter?.yAxisRight}
+                />
+              ) : null}
+              {isVertical ? (
+                <Fragment>
+                  <XAxis
+                    axisLine={false}
+                    domain={range}
+                    type="number"
+                    orientation="bottom"
+                    style={FONT_STYLE_VALUES}
+                    tickLine={false}
+                    tickCount={tickCount}
+                    xAxisId="bottom"
+                    tickFormatter={tickFormatter?.xAxisBottom}
+                  />
+                  <YAxis
+                    axisLine={false}
+                    dataKey="name"
+                    type="category"
+                    style={FONT_STYLE_CATEGORIES}
+                    tickLine={false}
+                    orientation="left"
+                    tickFormatter={(value: string) => labelMap?.[value] || value}
+                    // DO NOT SET yAxisId here
+                  />
+                </Fragment>
+              ) : null}
+              {isVerticalBiaxial ? (
+                <XAxis
+                  axisLine={false}
+                  domain={range}
+                  orientation="top"
+                  style={FONT_STYLE_VALUES}
+                  tickLine={false}
+                  tickCount={tickCount}
+                  type="number"
+                  xAxisId="top"
+                  tickFormatter={tickFormatter?.xAxisTop}
+                />
+              ) : null}
```

Comment:
if these are null, can you do `{isVerticalBiaxial && <Component/>}` then I don't think you need to pass the `null`

Reply:
Since there is no else statement here